api/src/main/java/io/grpc/InternalTimeUtils.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2024 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc;
+
+import java.time.Duration;
+
+@Internal
+public final class InternalTimeUtils {
+  public static long convert(Duration duration) {
```

Comment:
Code coverage is not detecting that this method is used from tests for some reason. I don't know if it is unable to detect static import of a function. Can you try removing the static import of InternalTimeUtils.convert in the test class, and import the class only, and use InternalTimeUtils.convert( in the test code and see if the Codecov warning goes away.

Reply:
No, it isn't the static import. The code coverage is powered by jacoco which instruments the bytecode to see when a line is executed.

What's probably happening here is it because called in the stub/ tests, but those tests don't instrument api/ classes. I wouldn't worry about it.